frontend/components/views/faq/Other.vue

```diff
@@ -99,6 +101,89 @@
         for more information.
       </v-expansion-panel-content>
     </v-expansion-panel>
+    <v-expansion-panel>
+      <v-expansion-panel-header>
+        <strong>What is your tagging policy?</strong>
+      </v-expansion-panel-header>
+      <v-expansion-panel-content>
+        <p>Artconomy has
+          <router-link :to="{name: 'Other', params: {question: 'content-policy'}}">very few restrictions</router-link>
+          on what content may be posted, but with this freedom comes a responsibility to tag content effectively.
+          Tagging content serves the following purposes:
+        </p>
+        <ul>
+          <li>It makes products, submissions, and characters easy for visitors to find via search. Customers interested
+            in your content may not be able to find it if it is not properly tagged.</li>
+          <li>It makes such content easy to <router-link :to="{name: 'Other', params: {}}">blacklist</router-link> for
+            those who do not wish to see it.</li>
+        </ul>
+        <p>
+          In sum, tagging allows viewers of the site to have control over their experience, and tagging allows artists
+          to find their audience. This, in combination with
+          <router-link :to="{name: 'Other', params: {question: 'content-policy'}}">content ratings</router-link>
+          allows Artconomy to serve a wide, often disparate range of content without telling artists what they may or
+          may not create.
+        </p>
+        <p>These purposes guide tagging-- which means that
+          <strong>appearances are more important than lore-based truth</strong> for content. This is commonly referred
+          to as a <strong>tag what you see</strong> policy. Tag what you see is the first priority, but you may also
```

Comment:
I realize this may mean that some characters are tagged in a way not true to their fully realized nature, but feel this is important since the platform's purpose is to help artists get sales, and people will be searching for products/submissions that match what they are looking to see above all else, and are more likely to commission based upon these features.